### PR TITLE
Cover/cmds/exp/ed: Add tests.

### PR DESCRIPTION
**Pull request by llogen**
Add tests.

Signed-off-by: llogen <christophlange95@googlemail.com>

### CODE REVIEW

**Comment by llogen** [comment]
WIP

File: cmds/exp/ed/filebuffer_test.go [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+	for _, tt := range testTableReadFile {
+		t.Run(tt.name, func(t *testing.T) {
+			err := tt.in.ReadFile(tt.methodin1, tt.methodin2)
+			if err != nil {
+				if err.Error() != tt.err.Error() {
+					t.Errorf("Expected value: %v, got: %v", tt.err, err)
+				}
+			}
+		})
+		defer os.Remove(tt.methodin2)
```
Comment (ChriMarMe) [inline cmds/exp/ed/filebuffer_test.go:592]: This does not work properly. File still exists after test.

File: cmds/exp/ed/filebuffer_test.go [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+	exp      *FileBuffer
+	err      error
+	methodin string
+}{
+	{
+		name:     "Test FileToBuffer: file exists",
+		in:       &FileBuffer{},
+		exp:      &FileBuffer{mod: true, file: []int{0}},
+		err:      fmt.Errorf(""),
+		methodin: CreateFile("filetobuffer"),
```
Comment (ChriMarMe) [inline cmds/exp/ed/filebuffer_test.go:609]: Same here

File: cmds/exp/ed/filebuffer_test.go (hunk 1 above)
Comment (walterchris) [reply cmds/exp/ed/filebuffer_test.go:592]: Probably use: https://pkg.go.dev/io/ioutil#example-TempFile ?

**Pull request closed without merging**